finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/commands/Sentinels.scala

```diff
@@ -0,0 +1,181 @@
+package com.twitter.finagle.redis.protocol.commands
+
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.redis.protocol.Commands._
+import com.twitter.finagle.redis.util._
+import com.twitter.finagle.redis.ClientError
+
+import java.net.InetSocketAddress
+
+import org.jboss.netty.buffer.ChannelBuffer
+import org.jboss.netty.buffer.ChannelBuffers
+
+case class SentinelMaster(name: ChannelBuffer)
+  extends Sentinel(SentinelMaster.channelBuffer, Seq(name))
+
+object SentinelMaster extends SentinelHelper {
+  val command = "MASTER"
+  def apply(args: Seq[Array[Byte]]): SentinelMaster = {
+    val name = trimList(args, 2, "SENTINEL MASTER")(0)
```

Comment:
abstract class Config(sub: ChannelBuffer, args: Seq[ChannelBuffer]) extends Command {
      // I think you're mentioning this one.
      def command = Commands.CONFIG
      def toChannelBuffer = RedisCodec.toUnifiedFormat(Seq(CommandBytes.CONFIG, sub) ++ args)
    }

    case class ConfigResetStat() extends Config(sub = ConfigResetStat.channelBuffer, args = Seq())
    object ConfigResetStat extends ConfigHelper {
      // rather than this one.
      val command = "RESETSTAT" // <-- Where is it used?
      def apply(args: Seq[Array[Byte]]): ConfigResetStat = new ConfigResetStat()
    }